hack/extract-release.sh

```diff
@@ -13,13 +13,13 @@ source "${OS_ROOT}/hack/common.sh"
 # Go to the top of the tree.
 cd "${OS_ROOT}"
 
-# Copy the linux release archives release back to the local _output/local/bin/linux/amd64 directory.
-# TODO: support different OS's?
+# Copy the release archives release back to the local _output/local/bin/... directories.
+# NOTE: On Mac and Windows you must pass WARN=1 in order to extract the output.
 os::build::detect_local_release_tars $(os::build::host_platform_friendly)
```

Comment:
Or maybe I misunderstood your question?

On Apr 21, 2016, at 6:55 PM, Jordan Liggitt <notifications@github.com>
wrote:

In hack/extract-release.sh
<https://github.com/openshift/origin/pull/8594#discussion_r60668218>:

> @@ -13,13 +13,13 @@ source "${OS_ROOT}/hack/common.sh"
>  # Go to the top of the tree.
>  cd "${OS_ROOT}"
>
> -# Copy the linux release archives release back to the local _output/local/bin/linux/amd64 directory.
> -# TODO: support different OS's?
> +# Copy the release archives release back to the local _output/local/bin/... directories.
> +# NOTE: On Mac and Windows you must pass WARN=1 in order to extract the output.
>  os::build::detect_local_release_tars $(os::build::host_platform_friendly)

how do you skip this behavior? every time I hit it, I haven't wanted it

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/8594/files/8439971d6186820daab14b829f6c84ce51728a45#r60668218>